lib/osf-components/addon/components/osf-layout/registries-side-nav/x-link/template.hbs

```diff
@@ -5,6 +5,7 @@
             {{if this.isCollapsed 'Collapsed'}}
             {{if this.isCurrentPage 'CurrentPage'}}
             {{if this.isDrawer 'Drawer'}}'
+        inert={{this.isInert}}
```

Comment:
`inert` applies to the entire subtree under the inert element -- it'd be cleaner to put it at the highest level in `osf-layout`, so you don't need to pass `leftClosed` down through all these layers